app/src/main/java/com/sopt/dive/screen/auth/SignInScreen.kt

```diff
@@ -0,0 +1,81 @@
+package com.sopt.dive.screen.auth
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.sopt.dive.R
+import com.sopt.dive.component.text.LabeledTextField
+import com.sopt.dive.component.button.SignButton
+import com.sopt.dive.component.text.Title
+import com.sopt.dive.ui.theme.DiveTheme
+
+@Composable
+fun SignInScreen(
+    id: String,
+    onIdChange: (String) -> Unit,
+    password: String,
+    onPasswordChange: (String) -> Unit,
+    onLoginClick: () -> Unit,
+    onSignUpClick: () -> Unit
+) {
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(horizontal = 40.dp, vertical = 40.dp),
```

Comment:
```suggestion
            .padding(40.dp),
```
이렇게 해도 똑같은 효과를 볼 수 잇다능 !!